recipes/micropip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "micropip" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/micropip-{{ version }}.tar.gz
+  sha256: 46e901e045860443808b9fd5ec7ad2ae7253ba25440c972b6d1d39e8b3f56f8a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+  run:
+    - python >=3.10
+    # From https://github.com/pyodide/pyodide/blob/main/packages/micropip/meta.yaml#L16
+    - packaging
+    - pyparsing
+    - pyodide-py
+
+test:
+  imports:
+    - micropip
+  commands:
+    - pip check
```

Comment:
Is there a `micropip check` command?

From https://github.com/conda-forge/staged-recipes/pull/20961#issuecomment-1299522331 :

> * [ ] BLD,TST: micropip: A `micropip check` command
>   
>   * https://pypi.org/project/pyodide-micropip-test/
> * [ ]  BLD,TST: meta.yml: add `micropip check` test command

Reply:
@westurner I don't think there is.

Reply:
@wolfv what would `micropip check` check?.